App/src/main/java/User.java

```diff
@@ -0,0 +1,76 @@
+
+
+public class User {
+	/*For the User class the fields needed are:
+	 * Name, age, city, state, disabilities, and
+	 * number of families.
+     * 
+     * Class also contains multiple methods for creating and sending the
+     * information of the user
+     * 
+	 * */
+	private String Name;
+	private Integer Age;
+    private Boolean disabilities;
+    private Boolean needMedAssist;
+    private Integer NumberOfFamilyMembers;
+    private Integer shelterNeedLength;
+    private static final int ADULT_AGE = 18;
```

Comment:
Suuper nitpicky optional style thing: I often like to put an extra space between sections, like having an empty line between your fields and your constructor, or between fields and constants, etc.